app/services/acuant/acuant_client.rb

```diff
@@ -29,26 +43,22 @@ def post_images(front_image:, back_image:, instance_id: nil)
       back_response = post_back_image(image: back_image, instance_id: instance_id)
       response = merge_post_responses(front_response, back_response)
 
-      check_results(response, instance_id)
+      results = check_results(response, instance_id)
+
+      if results.success?
+        pii = results.pii_from_doc
+        selfie_response = post_selfie(image: selfie_image, instance_id: instance_id)
+        [selfie_response, pii]
```

Comment:
Oof, yeah the PII is not on the selfie response, it's on the results 🤔. I wonder if there is a way to create a new response object or something like that that lets us get around having to return a tuple here?